AirCasting/Services/DownloadMeasurmentsService.swift

```diff
@@ -53,9 +53,14 @@ final class DownloadMeasurementsService: MeasurementUpdatingService {
         }
     }
     
-    private func updateForSession(uuid: SessionUUID) {
-        #warning("TODO: change last sync")
-        let syncDate = Date().addingTimeInterval(-100)
+    private func updateForSession(uuid: SessionUUID) {        
+        let session = try? persistenceController.viewContext.existingSession(uuid: uuid)
+        let lastMeasurementTime = session?.allStreams?
+            .compactMap { $0.lastMeasurementTime }
```

Comment:
super minor syntax tip: keypaths could be used here: `.compactMap(\.lastMeasurementTime)`